controllers/openshift_controller.go

```diff
@@ -936,13 +936,17 @@ func (r *KataConfigOpenShiftReconciler) processKataConfigInstallRequest() (ctrl.
 	if mcfgv1.IsMachineConfigPoolConditionTrue(foundMcp.Status.Conditions, mcfgv1.MachineConfigPoolUpdated) &&
 		foundMcp.Status.ObservedGeneration > r.kataConfig.Status.BaseMcpGeneration &&
 		foundMcp.Status.UpdatedMachineCount == foundMcp.Status.MachineCount {
-		r.Log.Info("set runtime class")
+		r.Log.Info("create runtime class")
 		r.kataConfig.Status.InstallationStatus.IsInProgress = "false"
-		return r.setRuntimeClass()
+		err := r.createRuntimeClass()
+		if err != nil {
+			return reconcile.Result{Requeue: true, RequeueAfter: 15 * time.Second}, err
```

Comment:
@bpradipt Sorry but I'm still confused here... this seems to change the behavior of `processKataConfigInstallRequest()` since it would have return `ctrl.Result{}, err` before this patch, right ? If so, this patch seems to be more than just changing the signature or I'm missing something(TM) ?

Reply:
Ah I see what you mean. I'll not add the "Requeue" params. I wanted to avoid an immediate reconciliation. I'll change this in a follow-on PR later on.